xivo_fetchfw/tests/test_storage.py

```diff
@@ -1,14 +1,12 @@
-# -*- coding: utf-8 -*-
-# Copyright 2013-2021 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2013-2022 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from __future__ import absolute_import
-import mock
 import os
 import tempfile
 import unittest
+from configparser import RawConfigParser
+from unittest.mock import Mock, MagicMock
```

Comment:
Import sorting :safety_vest:

Reply:
I'm all for import sorting, but if it blocks PRs we need to add isort ASAP, because it's annoying to do manually. 😅